include/cuco/bit_vector.cuh

```diff
@@ -59,51 +91,100 @@ class bit_vector {
   bit_vector();
   ~bit_vector();
 
-  void add(bool bit);  // adds a new bit at the end
-  void build();        // builds indexes for rank and select.
-
+  /**
+   * @brief adds a new bit at the end
+   *
+   * @param bit Boolean value of new bit to be added
+   */
+  void add(bool bit);
+
+  /**
+   * @brief Builds indexes for rank and select
+   *
+   * Also creates device-side snapshot
+   */
+  void build();
```

Comment:
Hmm, this also seems like specific for the trie implementation. Not sure if we should expose this in the generic public bit vector API.